docs/src/modules/java/pages/eventsourced.adoc

```diff
@@ -61,11 +61,11 @@ include::example$java-eventing-shopping-cart/src/main/java/shopping/cart/Shoppin
 
 == Emitting events
 
-Commands can modify state by emitting events. A command handler can emit an event by taking in a link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html[`CommandContext` {tab-icon}, window="new"] parameter, and invoking the link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html#emit(java.lang.Object)[`emit` {tab-icon}, window="new"] method on it. Invoking `emit` will immediately invoke the associated event handler for that event. This validates that the event can be applied to the current state, and also updates state for command handler processing.
+Commands can modify state by emitting events. A command handler can emit an event by taking in a link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html[`CommandContext` {tab-icon}, window="new"] parameter, and invoking the link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html#emit(java.lang.Object)[`emit` {tab-icon}, window="new"] method on it. 
 
-IMPORTANT: The **only** way for a command handler to modify Entity state is by emitting an event. Any modifications made directly to the state from the command handler are not persisted. When the Entity is passivated and reloaded, those modifications will not be present.
+When the command handler receives a request, the event is emitted and Akka Serverless immediately sends the event to the journal, and invokes the entity event handler.  Immediately after, the entity state is updated to the new value and can be used to create the reply. 
 
-The following example of a command handler emits an event:
+NOTE: An entity update event can occur before the event has been stored in the journal. This does not cause any concurrency issue because, Akka Serverless only process one command at a time per entity instance. The next command is not processed until an event from the previous command has been successfully been stored.
```

Comment:
As mentioned in the akkaserverless-docs PR, the original text makes more sense to me in this part. Maybe just leave out these particular changes.

The Java SDK is being changed and will have a different approach for emitting events and using the updated state.

Reply:
fixed: 500af37472fd58d0f872f30dcc80a12faf285798